sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/UDAQuerySuite.scala

```diff
@@ -354,6 +355,13 @@ abstract class UDAQuerySuite extends QueryTest with SQLTestUtils with TestHiveSi
         Row(3, 0, null, 1, 3, 0, 0, 0, null, 1, 3, 0, 2, 2) :: Nil)
   }
 
+  // SPARK-32159
+  test("array input types") {
```

Comment:
nit: `"SPARK-32159: array input types"` per http://spark.apache.org/contributing.html

Reply:
@erikerlandson I think you misunderstood the @HyukjinKwon comment. The correct one is `test("SPARK-32159: array input types") {`.

Reply:
I see - fixing it